utils/__snapshots__/linterError.test.js.snap

```diff
@@ -3,6 +3,6 @@
 exports[`Test utils/linterError.js Check formating of a error send to console log 1`] = `
 "[2m[utils/linterError.js] [0m[31m✖  [0m[31m
     my error[0m[0m
-    [2mLintError at linterError.test.js:12:10 
```

Comment:
This path looks wrong?

Reply:
It's correct it's ```const source = __filename;``` at linterError.test.js